### PR TITLE
Added request logging

### PR DESCRIPTION
**Pull request by shimshon-stratoscale**
@adir-stratoscale @Stratoscale/mgmt

### CODE REVIEW

File: hammock/common.py [hunk 1]
```diff
@@ -55,6 +56,16 @@ def log_exception(exc, request_uuid):
         logging.exception("[Internal server error %s]", request_uuid)
 
 
+def log_request(request_method, request_uri, resp_status, request_start):
+    duration_msec = (time.time() - request_start) * 1000
+    logging_method = logging.info
+    if request_method == 'GET' and 200 <= int(resp_status) <= 299:
```
Comment (maroun-stratoscale) [inline hammock/common.py]: +1 for not using the useless constant `GET = 'GET'`.

Comment (adir-stratoscale) [inline hammock/common.py]: maybe a safer way would be - `request_method.upper() == 'GET'`

File: hammock/common.py [hunk 2]
```diff
@@ -55,6 +56,16 @@ def log_exception(exc, request_uuid):
...
+        logging_method = logging.debug
+    logging_method("%(method)s %(route)s => returned %(retval)s in %(duration).2f msecs",
```
Comment (adir-stratoscale) [inline hammock/common.py]: consider call log `logging.log()` with the appropriate level. for example:
``` python
level = logging.DEBUG
logging.log(level, msg)
```

File: hammock/wrappers/wrapper.py [hunk 3]
```diff
@@ -85,9 +88,10 @@ def call(self, req):
             resp.update_content_length()
 
         except Exception as exc:  # pylint: disable=broad-except
-            self._resource.handle_exception(exc, self.exception_handler, req.uid)
+            self._resource.handle_exception(exc, self.exception_handler, req.uid, req.method, request_path, request_start)
         else:
             LOG.debug('[response %s] status: %s, content: %s', req.uid, resp.status, resp.content)
+            common.log_request(req.method, request_path, resp.status, request_start)
```
Comment (adir-stratoscale) [inline hammock/wrappers/wrapper.py]: maybe the call to `log_request` should be at the finally section.
this way you could remove the call to `log_request from the `handle_exception` function.

**Pull request merged**